src/views/ReturnDetails.vue

```diff
@@ -10,7 +10,15 @@
     <ion-content>
       <main>
         <ion-item lines="none">
-          <h1>{{ $t("Shipment ID") }}: {{ current.shipmentId }}</h1>
+          <ion-list class="ion-text-center">
+            <ion-title>{{ current.shopifyOrderName ? current.shopifyOrderName : current.hcOrderId }}</ion-title>
+            <!-- TODO: Fetch Customer name -->
+            <!-- <ion-label>{{ $t("Customer: <customer name>")}}</ion-label> -->
+          </ion-list>
+          <ion-item slot="end" lines="none">
```

Comment:
shouldn't use ion-item here. Instead we can use an ion-row here or normal div with css

Reply:
Done sir